Iterable/iterable-web-sdk
# Upgrade webpack-dev-server

## JIRA Ticket(s) if any

* [MOB-3220](https://iterable.atlassian.net/browse/MOB-3220)

## Description

Upgrades webpack-dev-server, so that we can get rid of the dependabot security issues with `glob-parent`

See migration guide from v3 to v4 here:

https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md

## Test Steps

1. Run `yarn clean` to remove all node modules for both the SDK and example repo
2. Run `yarn install:all && yarn start:all` and ensure you can go to http://localhost:8080 without any errors
